### ISSUE TITLE
Class 'MongoDB' not found

### ISSUE THREAD

**Issue by anned20** [bcosca/fatfree-core#177]
phpinfo() shows that MongoDB is correctly loaded.

**Comment by sn0opy** [comment]
The old MongoDB driver in PHP is gone and the new one is not yet supported in F3. 

https://github.com/bcosca/fatfree-core/issues/111

**Comment by anned20** [comment]
Ah. Any idea when it will be supported? Or a workaround?

**Comment by bcosca** [comment]
@ikkez can you stress-test the latest commit against cortex?

**Comment by ikkez** [comment]
the cortex tests still work fine within the old mongo extension and mongo 2.x.
I wasn't able to get the new mongodb extension to run in my php7 dev environment on my mac OS yet, due to some weird mac dependency issues, but I'll try to test this on a debian system soon as well.

**Comment by ptdev** [comment]
Hey there.
I've now tested these changes with both drivers and it seems to be working fine. That said, I've since upgraded my installations to php7 (was basically just waiting for this) and have been using mostly the new drivers.

This compatibility with the new mongodb driver should get included in the stable channel as a v3.6.1 to ease composer deployments on php7 environments without having to rely on dev-master as a repo.

Just a though.
And great job as usual.
cheers.
